src/index.ts

```diff
@@ -57,7 +58,22 @@ export interface SpannerOptions extends GrpcClientOptions {
   port?: number;
   sslCreds?: ChannelCredentials;
 }
-
+export interface RequestConfig {
+  client: string;
+  method: string;
+  reqOpts: {
```

Comment:
Shouldn't this be either any or a a generic type?

Reply:
currently added as `any` will see how this is used in other file and if we can make it generic than will do in subsequent PR.